src/queryValidator.js

```diff
@@ -0,0 +1,83 @@
+import { cmcdTypes, keyTypes, errorTypes } from './constants.js';
+import { createError } from './error.js';
+import checkQuotes from './utils/checkQuotes.js';
+
+const queryValidator = (queryString) => {
+  const error = [];
```

Comment:
Instead of creating the array inside the function, wouldn't it be better to pass it as a parameter?

Reply:
Yes, that's right. I changed it